packages/jobs/lib/activities.ts

```diff
@@ -430,7 +416,7 @@ export async function cancelActivity(workflowArguments: InitialSyncArgs | Contin
             nangoConnection,
             syncType,
             syncName,
-            activityLogId,
+            activityLogId: undefined,
```

Comment:
In case of action cancel we were passing the old activity log which was not creating an issue per say because we were not logging anything here but highlight the issue of this floating `activityLogId`